core/src/main/java/com/opensymphony/xwork2/ognl/OgnlUtil.java

```diff
@@ -261,7 +262,8 @@ public void setDevModeExcludedPackageExemptClasses(String commaDelimitedClasses)
     }
 
     private Set<String> parseExcludedPackageNames(String commaDelimitedPackageNames) {
-        Set<String> parsedSet = TextParseUtil.commaDelimitedStringToSet(commaDelimitedPackageNames);
+        Set<String> parsedSet = commaDelimitedStringToSet(commaDelimitedPackageNames)
+                .stream().map(s -> strip(s, ".")).collect(toSet());
```

Comment:
I think it's unintuitive to require a period at the of each excluded package. Currently if the period is missed, the exclusion will act like the regex `com\.package.*` which likely isn't what users configuring their application intend.

By stripping the period here, we retain backwards compatibility for excluded package lists that still have trailing periods.

However, we do drop support for wildcard package exclusions where applications genuinely desired the previous behaviour when a trailing period wasn't included. They will need to move such exclusions to the regex exclusion list instead.